app/scripts/deploy_service.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Deploys a Hivemind contract to localhost:8546
+ *
+ * Usage: node deploy.js <model_config file path> <deposit_amount>
+ */
+
+const chalk = require('chalk');
+const oasis = require('@oasislabs/client');
+const fs = require('fs');
+
+var multisig_deploy = oasis.workspace.Multisig.deploy(
+  ['b8b3666d8fea887d97ab54f571b8e5020c5c8b58', 'ff8c7955506c8f6ae9df7efbc3a26cc9105e1797'], 2,
```

Comment:
I don't think that you want to hard code addresses in the deployment script. I get what you're saying now about wanting deploy params from the CLI, but the easiest way to do this currently is to have node read `argv` and pass `oasis deploy -- arg1 arg2`